crawl-ref/source/spl-summoning.cc

```diff
@@ -3439,3 +3448,34 @@ int count_summons(const actor *summoner, spell_type spell)
 
     return count;
 }
+
```

Comment:
this needs more work since I need to make the mpr messages more generic and need to add back the logic to handle whether a non-player is the caster (in the original f. prism code, first commit in this PR).